nevergrad/benchmark/plotting.py

```diff
@@ -187,6 +187,11 @@ def create_plots(df: pd.DataFrame, output_folder: PathLike, max_combsize: int =
             # save
             name = "fight_" + ",".join("{}{}".format(x, y) for x, y in zip(fixed, case)) + ".png"
             name = "fight_all.png" if name == "fight_.png" else name
+            if name == "fight_all.png":
+                with open(str(output_folder / name) + ".cp.txt", "w") as f:
+                   f.write("ranking:\n")
+                   for i, algo in enumerate(data_df.columns[:8]):
+                       f.write("  algo {i}: {algo}\n")
```

Comment:
do you need the "algo" and the number? it does feel weird to have a list
 algo 1 : XXXX
 algo 2: YYYYY

I would go for just an ordered list, or if need be the number as well, but "algo" should be assumed